hca_ingest/api/ingestapi.py

```diff
@@ -185,11 +185,11 @@ def get_user_projects(self):
         return self.get_all(user_project_url, 'projects')
 
     def get_entity_by_uuid(self, entity_type, uuid):
-        url = self.url + f'/{entity_type}/search/findByUuid?uuid=' + uuid
+        url = f'{self.url}/{entity_type}/search/findByUuid?uuid=' + uuid
 
         # TODO make the endpoint consistent
         if entity_type == 'submissionEnvelopes':
-            url = self.url + f'/{entity_type}/search/findByUuidUuid?uuid=' + uuid
+            url = f'{self.url}/{entity_type}/search/findByUuidUuid?uuid=' + uuid
 
         return self.get(url).json()
```

Comment:
good idea.
Even cleaner would be to use `params` argument for `requests.get()`

Reply:
done and pushed